aws/resource-count-aws.sh

```diff
@@ -68,6 +68,39 @@ else
    WITH_DATA="false"
 fi
 
+
+##########################################################################################
+## Optionally pass an AWS profile to use. Profiles are available at ~/.aws/config (Linux & Mac) or %USERPROFILE%\.aws\config (Windows)
+##########################################################################################
+
+echo ">>> Profiles are available at ~/.aws/config (Linux & Mac) or %USERPROFILE%\.aws\config (Windows)"
+
+ORIGINAL_AWS_PROFILE_ENV=$(printenv AWS_PROFILE)
```

Comment:
This variable stores the current AWS_PROFILE, but is never used to restore that profile after the script exits.